frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card-utils.ts

```diff
@@ -49,7 +49,11 @@ const getMaxVal: GetMaxVal = (response, humanize) => {
   const result: PrometheusResponse['data']['result'] = _.get(response, 'data.result', []);
   let maxVal = { unit: '', value: 0, string: '' };
   if (result.length) {
-    maxVal = humanize(_.maxBy(result, (r) => Number(r.value[1])).value[1]);
+    maxVal = humanize(_.maxBy(result, (r) => Number(r.value[1])).value[1]) as {
+      unit: string;
+      value: number;
+      string: string;
+    };
```

Comment:
Don't use `as`. It's better to use the `PrometheusResponse` type. We can break that up into a separate type for data points if needed.